packages/protocol/script/SetDcapParams.s.sol

```diff
@@ -0,0 +1,89 @@
+// SPDX-License-Identifier: MIT
+//  _____     _ _         _         _
+// |_   _|_ _(_) |_____  | |   __ _| |__ ___
+//   | |/ _` | | / / _ \ | |__/ _` | '_ (_-<
+//   |_|\__,_|_|_\_\___/ |____\__,_|_.__/__/
+//
+//   Email: security@taiko.xyz
+//   Website: https://taiko.xyz
+//   GitHub: https://github.com/taikoxyz
+//   Discord: https://discord.gg/taikoxyz
+//   Twitter: https://twitter.com/taikoxyz
+//   Blog: https://mirror.xyz/labs.taiko.eth
+//   Youtube: https://www.youtube.com/@taikoxyz
+
+pragma solidity 0.8.24;
+
+import "forge-std/Script.sol";
+import "forge-std/console2.sol";
+
+import "../contracts/thirdparty/automata-attestation/AutomataDcapV3Attestation.sol";
+import "../contracts/L1/verifiers/SgxVerifier.sol";
+import "../test/automata-attestation/utils/DcapTestUtils.t.sol";
+import "../test/automata-attestation/utils/V3JsonUtils.t.sol";
+
+contract SetDcapParams is Script, DcapTestUtils, V3JsonUtils {
+    uint256 public ownerPrivateKey = vm.envUint("PRIVATE_KEY"); // Owner of the attestation contract
+    address public dcapAttestationAddress = vm.envAddress("ATTESTATION_ADDRESS");
+    address public sgxVerifier = vm.envAddress("SGX_VERIFIER_ADDRESS");
+    string public tcbInfoPath = vm.envString("TCB_INFO_PATH");
+    string public idPath = vm.envString("QEID_PATH");
+    string public v3QuotePath = vm.envString("V3_QUOTE_PATH");
+    bytes32 public mrEnclave = vm.envBytes32("MR_ENCLAVE");
+    bytes32 public mrSigner = vm.envBytes32("MR_SIGNER");
+
+    function run() external {
+        require(ownerPrivateKey != 0, "PRIVATE_KEY not set");
+        require(dcapAttestationAddress != address(0), "ATTESTATION_ADDRESS not set");
+
+        vm.startBroadcast(ownerPrivateKey);
+
+        // all in one
```

Comment:
What does this mean? :)

Reply:
it calls 5 functions in run() for register platform, set MrEnclave, and set sgx instance id. for test purpose it's ok, for real use case, maybe we should call individual function instead.